core/spec.md

```diff
@@ -1117,6 +1122,42 @@ HTTP body.
 
 ---
 
+### Registry Well-Known Config
+
+The registry comes with a well known location `/.well-known/xregistry.json`,
+following [RFC8615](https://datatracker.ietf.org/doc/html/rfc8615).
+
+It acts as a single entrypoint from where locations and capabilities of the
+xRegistry implementation can be retrieved. Since this URL is always the same,
+it can be used to find out whether xRegistry is supported and where to find the API.
+It also allows to support multiple versions of the xRegistry protocol at the same time
+and allows implementers to choose their own API base path.
+
+The capabilities described on this level are independent from the xRegistry protocol version or its current meta model.
+They resemble the capabilities of the xRegistry implementation itself that consumers can rely upon.
+
+```yaml
+{
+  # Some high-level description of the xRegistry that are independent from version of the protocol nor its meta model
+  "description": "STRING", ?
+  "documentation": "URL", ?
+  "labels": { "STRING": "STRING" * }, ? #
+
+  # Describe where to find the xRegistry API and its capabilities
+  "apis": [{ # Major version of the xRegistry API protocol. In the future, there could be v2 in parallel.
+    "specversion": "STRING", # full specversion, as returned by the API root response
+    "apiurl": "URL", ? # Point to root URL of xRegistry API, e.g. /x-registry/v1/
+    "capabilities": ["write", "update", "inline", "filter", "..."] # TODO: TBD
+    "modelurl": "URL", ? # Point to registry meta model
```

Comment:
"modelurl" seems like it would be a duplicate of the `/model` API that's part of the API itself. no?
Or do you see cases where there's no "apiurl" but still a "modelurl" ?
I guess it depends on whether the "files" stuff that we've talked about includes the model definition or whether it's just the metadata.

Reply:
yes. I actually wasn't really sure if we should include it or not. I put it in, so it can be discussed.